db/migrate/20190521214615_rename_url_to_website_on_provider.rb

```diff
@@ -0,0 +1,5 @@
+class RenameUrlToWebsiteOnProvider < ActiveRecord::Migration[5.2]
+  def change
+    rename_column :provider, :url, :website
```

Comment:
Isn't there a risk that with a change like this, the C# app might have a fit?

Reply:
Yes :(

Reply:
I have another idea... watch this space (tomorrow morning)

Reply:
Solved